tools/ci/docker/linux/Dockerfile

```diff
@@ -191,6 +191,14 @@ RUN cd renesas-tools/source/newlib && \
 RUN cd /tools/renesas-tools/build/gcc && \
   make && make install
 
+# Install Rust and targets supported from NuttX
```

Comment:
This is being installed in the Renassas build image. It should be done in its own image.